src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -195,20 +195,23 @@ ValidatorList::load(
 }
 
 boost::filesystem::path
-ValidatorList::GetCacheFileName(PublicKey const& pubKey)
+ValidatorList::GetCacheFileName(
+    ValidatorList::unique_lock const&,
+    PublicKey const& pubKey)
```

Comment:
Super nitpicky: const?

Reply:
Looks like I already made that change in #3619, and this is already merged, so I'll let it go from there.